python/syntax.lark

```diff
@@ -45,7 +45,9 @@ anonymous_func: "(" arguments "->" NAME ":" (instruction (";" instruction)*) ")"
 char: "\\" ( escape_code | "{" /./ "}")
 escape_code: /[tnr]/
 tupledef: "(" NAME ("," NAME)* ")"
+listdef: LIST "[" NAME "]"
 tupleval: "(" value ("," value)* ")"
+listval: "[" (value ("," value)*)? "]"
```

Comment:
Perhaps we would want to add support for a trailing comma? Also, should we use commas or semicolons/newlines to separate tuple/list items?

Reply:
commas for now, also no trailing commas, i dont like them

Reply:
Oof, trailing commas are good for Git diffs though; otherwise you end up having diffs like

```diff
  a,
- b
+ b,
+ c
```

instead of

```diff
  a,
  b,
+ c,
```